asb-ballerina/Module.md

```diff
@@ -1,473 +1,262 @@
+* Create messaging entity, such as a queue, topic or subscription.
   If you don't have these items, learn how to
     * [Create a queue in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal)
     * [Create a Topic using the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-a-topic-using-the-azure-portal)
     * [Create Subscriptions to the Topic](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-subscriptions-to-the-topic)
 
-* Java 11 Installed
-  Java Development Kit (JDK) with version 11 is required.
-
-* Ballerina SLAlpha5 Installed
-  Ballerina Swan Lake Alpha 5 is required.
+* Obtain tokens
 
-* Shared Access Signature (SAS) Authentication Credentials
+    Shared Access Signature (SAS) Authentication Credentials are required to communicate with the Azure Service Bus.
     * Connection String
     * Entity Path
 
-## Configuration
-Instantiate the connector by giving authorization credentials that a client application can use to send/receive messages
-to/from the queue/topic/subscription.
-
-### Getting the authorization credentials
+    Obtain the authorization credentials:
+    * For Service Bus Queues
 
-#### For Service Bus Queues
+        1. Make sure you have an Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/en-us/free/) before you begin.
 
-1. Make sure you have an Azure subscription. If you don't have an Azure subscription, you can create a
-   [free account](https://azure.microsoft.com/en-us/free/) before you begin.
+        2. [Create a namespace in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-namespace-in-the-azure-portal)
 
-2. [Create a namespace in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-namespace-in-the-azure-portal)
+        3. [Get the connection string](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string)
 
-3. [Get the connection string](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string)
+        4. [Create a queue in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal). It is in the format ‘queueName’.
 
-4. [Create a queue in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal). 
-   It is in the format ‘queueName’.
+    * For Service Bus Topics and Subscriptions
```

Comment:
Please review this section with the comments added to the section above.

Reply:
Resolved and amended the changes. Thank you for the suggestion.